### DISCUSSION TITLE
Can't remove default login (running locally)

### DISCUSSION THREAD

**Discussion by xjmzx** [go-shiori/shiori#800] [Q&A]
Hello. I may have missed some documentation on how to remove the default login.
I am only running locally on my computer (as that's how I want to run the service) but after creating a new account (non-visitor) I don't see any option for owner, consequently, I can only see that I can make edits by logging in with the default account, not the newly created one. Am I doing something wrong or is this simply a limitation?

I think I'm running the latest version.

**Comment by Monirzadeh** [comment]
> Hello. I may have missed some documentation on how to remove the default login. I am only running locally on my computer (as that's how I want to run the service) but after creating a new account (non-visitor) I don't see any option for owner, consequently, I can only see that I can make edits by logging in with the default account, not the newly created one. Am I doing something wrong or is this simply a limitation?
> 
> I think I'm running the latest version.

i can't reproduce that.
any specific setup do you use? i can easily longing with `shiori` account and create new account (for example `newUser` as owner)
when i login `newUser` i can remove `shiori` and use `newUser` instead of that.
can you clone main repository and run shiori with
```
make run-server
```
do you still have this problem?

**Comment by fmartingr** [comment]
With "latest" are you referring to `1.5.5` or the latest one from github?

**Comment by xjmzx** [comment]
Sorry for the late reply. 

Seems I was able to get the new admin user (owner) working the next time I tried. I'm not sure which version I was running (I cloned the gh repo and then ran `go build` 

Can't seem to find a way to print the version. Anyway my bad. I wasn't 100% it was a bug.  Please put it down to user error!

**Reply by xjmzx** [reply]
To reconfirm, the default user is no longer there. Just the newly created user.
Thanks for the reply. I'll close this.

**Reply by fmartingr** [reply]
Thanks for confirming, better be safe than sorry!
